<ui version="4.0" >
 <class>VP1PluginDialog</class>
 <widget class="QDialog" name="VP1PluginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_selectplugin" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Select plugin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_plugins" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimumContentsLength" >
          <number>50</number>
         </property>
         <item>
          <property name="text" >
           <string>No plugins loaded</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_fullpathtitle" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Full path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fullpath" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>n/a</string>
         </property>
         <property name="textInteractionFlags" >
          <enum>Qt::TextSelectableByMouse</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_channels" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Channels available in this plugin:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QTextBrowser" name="textBrowser_channels" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="textInteractionFlags" >
          <enum>Qt::TextSelectableByMouse</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_unload" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Unload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>VP1PluginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>355</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
